lib/cocoapods/installer.rb

```diff
@@ -604,6 +605,20 @@ def write_pod_project
       end
     end
 
+    # Shares schemes of development Pods.
+    #
+    # @return [void]
+    #
+    def share_development_pod_schemes
+      development_pod_targets = sandbox.development_pods.keys.map do |pod|
+        pods_project.targets.select { |target| target.name =~ /^Pods-.*-#{pod}$/ }
+      end.flatten
+
+      development_pod_targets.each do |pod_target|
+        Xcodeproj::XCScheme.share_scheme(pods_project.path, pod_target.name, ENV['USER'])
```

Comment:
This does share the scheme from the user's directory to the common one - if the last argument is optional, we can drop it :tada: